<launch>

  <node name="joy_node" pkg="joy" type="joy_node" />

  <node name="xbox_to_twist" pkg="moveit_experimental" type="xbox_to_twist" output="screen" />

  <node name="jog_arm_server" pkg="moveit_experimental" type="jog_arm_server" output="screen" >
    <param name="parameter_ns" type="string" value="jog_arm_server" />
    <rosparam command="load" file="$(find moveit_experimental)/jog_arm/config/jog_settings.yaml" />
  </node>

</launch>
